src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -102,10 +107,14 @@ public void parse_repeatedNonTagValue_failure() {
         assertParseFailure(parser, INVALID_EMAIL_DESC + validExpectedPersonString,
                 Messages.getErrorMessageForDuplicatePrefixes(PREFIX_EMAIL));
 
-        // invalid phone
+        // invalid phone - wrong characters
```

Comment:
Was a bit confused with "wrong characters" but realised this meant "phone number has characters (and not just numbers)". Maybe a slight rephrase would be good?